firefox_puppeteer/tests/test_page_info_window.py

```diff
@@ -68,6 +68,11 @@ def opener(win):
                            opener,
                            )
 
+        if self.marionette.session_capabilities['platform'] == 'WINNT':
+            self.assertRaises(ValueError, self.browser.open_page_info_window,
+                              trigger='shortcut')
```

Comment:
Please move this if block into the for loop and continue with the next trigger if the conditions are met.